flash/audio/source_separation/__init__.py

```diff
@@ -0,0 +1,2 @@
+from flash.audio.source_separation.data import AudioSourceSeparationData
+from flash.audio.source_separation.model import AudioSourceSeparator
```

Comment:
Should we simplify it for `AudioSeparator` ?

Reply:
Maybe just `SourceSeparator` . Because its already inside the audio domain ?

Reply:
For consistency, we keep the data type in the task name as prefix.

noob question: Mind explaining me what source mean in this context. I believe it could be confusing for new users.

Reply:
Yes I understand. `Source` is analogous to `Speaker` in the an audio file which has lets say multiple speakers speaking simultaneously. So the speech for multiple speakers are isolated for individual speakers and separated.